doc/development.md

```diff
@@ -63,6 +63,12 @@ HELPERS=HOST ./build_blackbox_tests
 sudo sh -c 'PATH=$PWD/bin/amd64:$PATH ./tests.test'
 ```
 
+To run a subset of the blackbox tests, pass a regular expression into `-test.run`. As an example:
+
+```
+sudo sh -c 'PATH=$PWD/bin/amd64:$PATH ./tests.test -test.run TestIgnitionBlackBox/Preemption.*'
```

Comment:
Might add that you can get a list of tests with `-list`, otherwise LGTM.

Reply:
:+1: - adding this in